env.template

```diff
@@ -19,6 +19,7 @@ EMAIL_PASSWORD=example password
 EMAIL_URL=http://localhost:XXXX
 EMAILS_ENABLED=true
 SCHEDULE_JOB=1 0 1 * * *
+LOADER_QUEUE=crdcdh-queue
 
```

Comment:
For Node.js AWS SDK to load ~/.aws/config, need add an environment variable here as listed below:
AWS_SDK_LOAD_CONFIG=1